app/components/close_help_post.py

```diff
@@ -90,6 +90,17 @@ async def solved(
             title_prefix = additional_reply = None
         await self.close_post(interaction, "solved", title_prefix, additional_reply)
 
+    @solved.autocomplete("config_option")
+    async def option_autocomplete(
+        self, _: dc.Interaction, current: str
+    ) -> list[app_commands.Choice[str]]:
+        docs = cast("Docs", self.bot.cogs.get("Docs", SimpleNamespace(sitemap={})))
```

Comment:
Can we not just do:

```suggestion
        if not (docs := cast("Docs | None", self.bot.cogs.get("Docs"))):
            return []
```

It reads nicer than the `SimpleNamespace` in my opinion.